relentless/potential/potential.py

```diff
@@ -543,15 +592,34 @@ def __init__(self, nbins, rmin, rmax, fmax=None, fcut=None, edges=True):
         else:
             self._r = rmin + self._dr*(np.arange(nbins, dtype=np.float64)+0.5)
 
+        self._shift = shift
```

Comment:
Let's make this publicly accessible as `self.shift`. It's OK if the user wants to switch it between True or False directly.